Update font awesome instructions when using Kits

gregveres opened the pull request:
I have added information on using Font Awesome Pro Kits. In my opinion, this is the way to go when using Font Awesome Pro since it allows you to define a collection of icons that only include icons you use in your project. 

I also included a couple of other tips I wish were written down when I was learning how to use Font Awesome in Quasar. 

If this information was in this doc, it would have saved me at least  3/4 of a day figuring it all out for myself and pestering people on discord.

<!--
  Please make sure to read the Pull Request Guidelines:
  https://github.com/quasarframework/quasar/blob/dev/CONTRIBUTING.md#pull-request-guidelines
-->

<!-- PULL REQUEST TEMPLATE -->
<!-- Update "[ ]" to "[x]" to check a box (space sensitive) -->

**What kind of change does this PR introduce?** <!-- Check at least one -->

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** <!-- Check one -->

- [ ] Yes
- [x] No

<!-- If yes, please describe the impact and migration path for existing applications: -->

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch (or `v[X]` branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on an Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) <!-- for faster update click on "Suggest an edit on GitHub" at bottom of page --> or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to [start a new feature discussion](https://github.com/quasarframework/quasar/discussions/new?category=ideas-proposals) first and wait for approval before working on it)

**Other information:**

gregveres committed:
Update font awesome instructions when using Kits

I have added information on using Font Awesome Pro Kits. In my opinion, this is the way to go when using Font Awesome Pro since it allows you to define a collection of icons that only include icons you use in your project. 

I also included a couple of other tips I wish were written down when I was learning how to use Font Awesome in Quasar. 

If this information was in this doc, it would have saved me at least  3/4 of a day figuring it all out for myself and pestering people on discord.

rstoenescu reviewed (changes requested):
Hi,

Thank you for contributing!
I think it would be better if the two ways are separated. Some of the steps are not required when using Kits, so it'll only confuse someone reading the guide.

Can you please make a separate guide for the Kits?

gregveres commented:
Should it be a different page, or a different guide on the same page (say below the existing Font Awesome Pro section)?

gregveres committed:
split FA-pro kit and non-kit instructions

As requested, I have split the kit instructions out as a seperate section to avoid any confusion and make the instructions more clearer.

gregveres commented:
@rstoenescu I have updated the document file to split out the Kit instructions. There is now a section for non-kit installation and a section for kit installation.

rstoenescu commented:
Hi Greg, thank you again for putting this together, and apologies that it sat for so long. The docs page has since moved on to Fontawesome v7, so rather than asking you to rebase a two-year-old branch, we've adapted your material to the current page and landed it on dev in a3af26773: the Kits section, the whole-icon-set replacement example, plus a note that a subsetted kit must include the Light style icons the `fontawesome-v7-pro` icon set references. The npm token / `@fortawesome/fontawesome-pro` install steps were dropped from the Kits flow since a self-hosted kit download does not need them. Closing this one since the content is now in the docs. Your write-up saved the next person that 3/4 of a day you mentioned, much appreciated!
